shared/modelUtils.js

```diff
@@ -5,6 +5,8 @@
  */
 var BaseModel = require("./base/model"),
     BaseCollection = require("./base/collection");
+    sanitizer = require("sanitizer");
```

Comment:
Looks like you added another dependency `sanitizer`, but it isn't included in the package.json? I'm also curious if there is a way we can do this without adding another dependency here.

There is something that we added to the change.org app that is doing this data sanitization that we haven't had time to port back to Rendr itself.  I can take a look to see what is going on there (it's been a while).